front/lib/connector_providers.ts

```diff
@@ -420,7 +420,7 @@ export const CONNECTOR_CONFIGURATIONS: Record<
       selected: "read",
       unselected: "none",
     },
-    isDeletable: true,
+    isDeletable: false,
```

Comment:
Only change in front - since it's not a remote db anymore we do not allow deletion.